### ISSUE TITLE
Easynews

### ISSUE THREAD

**Issue by brianamit2017** [danamag/stremio-addons-list#167]
### Addon Manifest URL

https://ea627ddf0ee7-easynews.baby-beamup.club/manifest.json

### Addon Description

Stream movies and series straight from Easynews web search!

### Language of Content

None

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [X] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [X] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [X] tv shows
- [ ] video games

**Comment by Ramghh** [comment]
I don't know what this addon does but it is not working.

**Comment by mhmatthewhugley** [comment]
Works perfectly. Tested with trial account.

**Comment by sylverstream** [comment]
Having issues with buffering. Tried many different streams. Downloading from easynews directly is fast enough.

**Comment by FiefNoodles** [comment]
great addon

**Comment by Artus13** [comment]
Cannot add this addon to stremio, i'm getting the error : " ERR_OPENING_MEDIA failed to fetch"

**Comment by GruNostalgia** [comment]
I am unable to get this add on installed onto stremio. I get serialization errors 1 and 2. When i try it install via android i get error occured while adding add on.

**Comment by GruNostalgia** [comment]
> great addon

How did you get install it? Desktop, mobile, TV?

**Comment by icpryde** [comment]
works great, happy i can combine both EN and PM for my sources, always install addons on your computer not from your phone or android tv.

**Comment by sasagr** [comment]
I was very happy to see this addon, but I got a bit disappointed noticing that the results are poor or sometimes null, especially when it comes to Movies. I'm not sure if I'm doing something wrong (not that I can do much on the addon), but I have noticed I get zero or very limited results compared to what I can find doing a simple web search on Easynews website. Just a few examples: Oppen...heimer (zero results in the addon), The Crea...tor (zero results in the addon), MI Dead Reckoning (only one 1080p result). Their search websites show many results, including 2160p resolutions. The TV show results instead seem to be more rich and maybe it does not need any adjustment.
Is this a limit of the addon or am I missing something?

**Comment by tundrablasen** [comment]
i just wanted to thank the @brianamit2017 for the Easynews Addon. Its great to have the EN service now on the stremio plattform. i daily driving this addon and it works well. i rarely miss any content. so thank you very much that you give us choice and please continue to maintain the addon in foreseeable future.

**Comment by sasagr** [comment]
I agree. After the recent updates, the addon works perfectly. An applause to the developer @brianamit2017

**Comment by tundrablasen** [comment]
> I agree. After the recent updates, the addon works perfectly. An applause to the developer @brianamit2017

giving a thumbs up on OPs Post here will get synced to the stremio community addons website and give Easynews more visibility.

**Comment by sasagr** [comment]
> > I agree. After the recent updates, the addon works perfectly. An applause to the developer @brianamit2017
> 
> giving a thumbs up on OPs Post here will get synced to the stremio community addons website and give Easynews more visibility.

done!

**Comment by IrlJidel** [comment]
Love this addon - many thanks.

Would it be possible to add support for configuring it to exclude certain resolutions?

**Comment by Kwaice** [comment]
Thanks for the add-on.

I can't seem to get any results while searching for "Demain nous appartient" (french TV show).

The TV show is present on easynews. Any idea how I can watch it ?

Thank you.

**Comment by znre** [comment]
I’m experiencing some problems recently, it seems that the add on can’t play any videos even though my speed in EN’s route selector is 70 Mbps+

I reckon the route in the add on is defaulting to the automatic route instead of the one selected at EN settings?

I can download files just fine and plenty fast when using EasyNews with a web browser. I can also stream Easynews perfectly when using Kodi and FEN Light

**Comment by terkeitel** [comment]
This addon doesn't work. I will be happy to contribute financially towards making it stable and working perfectly.

**Comment by znre** [comment]
The addon doesn't work now. It gets stuck on loading.

Can't reinstall it either.

![image](https://github.com/danamag/stremio-addons-list/assets/70197545/8cf7acfe-1567-46f0-b987-ff0ff064e388)
